internal/server/place/golden/get_locations_ranking/california.json

```diff
@@ -5,292 +5,292 @@
         "info": [
           {
             "rank": 1,
-            "value": 9943046,
+            "value": 10039107,
```

Comment:
This is from PEP 2019, and though we have 2020 data from ACS 5 year and Decennial. Previous number came from previous PEP import (PEP, P1Y). According to that, population did drop from 2019 to 2020.

https://autopush.datacommons.org/browser/geoId/06037?statVar=Count_Person

Reply:
Probably because of the ranking here: https://github.com/datacommonsorg/mixer/blob/master/internal/server/ranking/ranking.go#L44-L46 ?
@shifucun 

India folks are updating PEP to have 2020.

Reply:
Can you update the ranking to be below and see it's getting better:

"CensusPEP": {{MM: s("CensusPEPSurvey"), OP: s("P1Y")}: 0}, // Population

Reply:
There are two CensusPEPSurvey import now: https://autopush.datacommons.org/browser/geoId/06?statVar=Count_Person

And we should prefer the one with "P1Y". @pradh after the data is in, I guess we can remove the other import?

Reply:
Ah, that update is in.  So yes P1Y SG.

Reply:
@Spaceenter @shifucun should we prefer a more recent date for this if available?

Reply:
yes, the new CensPEPSurvey + P1Y would have 2020 data as well.